reporter-web-app/src/components/TableView.js

```diff
@@ -194,6 +194,8 @@ class TableView extends React.Component {
                     </span>
                 ),
                 responsive: ['md'],
+                sorter: (a, b) => a.declaredLicenses.size - b.declaredLicenses.size,
```

Comment:
Commit message: Please add a dot after "Closes #2606" and a blank line to separate it from the footer.

Reply:
Fixed.